src/LiveScanClient/azureKinectCapture.cpp

```diff
@@ -102,10 +155,23 @@ bool AzureKinectCapture::AcquireFrame()
 		return false;
 	}
 
+	//We need to resize the color image, so that it's height fits the depth camera height, while preserving the aspect ratio of the color camera
+
+	//Convert the k4a_image to an OpenCV Mat
+	cv::Mat cImg = cv::Mat(k4a_image_get_height_pixels(colorImage), k4a_image_get_width_pixels(colorImage), CV_8UC4, k4a_image_get_buffer(colorImage));
+
+	//Resize the k4a_image to the precalculated size. Takes quite along time, maybe there is a faster algorithm?
+	cv::resize(cImg, cImg, cv::Size(color_image_downscaled_width, color_image_downscaled_height), cv::INTER_NEAREST);
```

Comment:
Using INTER_NEAREST may lead to fairly poor quality. I would suggest using INTER_LINEAR as a reasonable speed/quality tradeoff.

Reply:
How badly would changing to INTER_LINEAR impact the performance?